convert2rhel/pkghandler.py

```diff
@@ -66,6 +66,14 @@ def call_yum_cmd_w_downgrades(cmd, fingerprints):
         # handle error condition
         loggerinst.info("Resolving dependency errors ... ")
         resolve_dep_errors(output, [])
+        # if we have problematic packages, remove them
+        problematic_pkgs = get_problematic_pkgs(output, [])
+        if problematic_pkgs["errors"]:
+            loggerinst.warning(
+                "Removing problematic packages to continue with conversion: \n%s" %
+                "\n".join(problematic_pkgs["errors"])
+            )
+            utils.remove_pkgs(problematic_pkgs['errors'], critical=False)
```

Comment:
@bocekm do we want to ask the user to continue here? Or is a warning good enough

Reply:
At this stage of conversion there's no point in asking them because it's already past the point of no return and answering no would make their system partially converted with no easy way to get to the pre-conversion state. So, warning is enough.